src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Sort.cs

```diff
@@ -210,6 +210,9 @@ public static string GetSortComparatorIdForType(DType type)
 
             switch (type.Kind)
```

Comment:
Not necessarily as part of this change, but this should be pulled back to PA-Client, it only affects JSTranslator's impl of Sort/SortByColumns.

Reply:
Included it as part of this change and added it to PA-Client pull request

Reply:
Thanks!